src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,32 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+            display: inline-block;
+            background-color: rgb(242, 242, 242);
+            padding: 12px 16px;
+            margin: 8px 0px;
+        }
+    </style>
 </head>
 <body>
-    <h4>You have logged into Microsoft Azure!</h4>
-    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 10 seconds.</p>
+    <h3>You have logged into Microsoft Azure!</h3>
+    <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 2 minutes.</p>
+    <h3>Announcements</h3>
+    <p>[Windows only] Starting in May 2023, Azure CLI will authenticate using the <a href="https://learn.microsoft.com/en-us/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker by default.</p>
```

Comment:
```suggestion
    <p>[Windows only] Starting in May 2023, Azure CLI will authenticate using the <a href="https://learn.microsoft.com/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker by default.</p>
```

Reply:
@medhir can you give a high-level view of the benefits of WAM here so that people do not have to open an additional link to learn about WAM.